feat(content): Make decline choices in FW prologue more direct (#10757)
endless-sky/endless-sky: data/human/free worlds 0 prologue.txt

			choice
				`	"Sure, I'd be glad to do that."`
					accept

> 				`	"After the way the last captain responded to me scanning him, I'm not risking my head for the Free Worlds again."`

Author: I think it's reasonable that the player character could be annoyed enough at JJ to not bother with the niceties. Not mentioning the possibility of the Cruiser chatting you up after a scan is a pretty big oversight on his part.

> 				`	"After the way the last captain responded to me scanning him, I'm not risking my head for the Free Worlds again."`

Reviewer: Maybe. But I think either reaction would be reasonable. After all, JJ had no way of knowing that the Cruiser captain was getting paid to advertise their engines. I can see how some people might be that annoyed, but I don't think the captain I would personally play would be.

Maybe having two different options that both decline would work here, or fleshing out this conversation a little bit more to touch on how the Cruiser hailed you before outright denying the mission.

			`When you walk into the bar, Tomek looks very glad to see you. "Here's the deal," he says. "One of our freighters, the <npc>, was just disabled in the <waypoints> system. We need you to fly there immediately, repair it, and bring it back here before it can be captured by pirates. Do you think you're up to the task?"`
			choice
				`	"Sure!"`

> 				`	"Escort duty was one thing, but now you're ordering me around like you're my superior? I'm out."`

Reviewer: This seems overly aggressive. Tomek is offering you a job, and I don't think it reads like he's giving you a direct order.
```suggestion
				`	"Sorry, no. I'm just a merchant captain. Find someone else to do combat missions."`
```

Author: It's also not a combat mission though. Tomek's brief doesn't say you'll have to fight off the pirates to save the Shackleton, just that you have to repair it and escape (which is usually what happens in practice). There's more combat in the previous two escort jobs than in this mission, so why would the player character turn down this mission for "combat" reasons at this point?

Reviewer: Whoops, that's a good point. Still not sure the superior thing works though...

Maybe:
```suggestion
				`	"Sorry, no. I'm a merchant captain with bills to pay. I'm not going to go search for your missing ships in the middle of nowhere."`
```

				`	"That sounds like a terrible idea."`
					goto terrible
			
			`	Eyes laughs, spitting out his milkshake in the process. "We won't be asking you to do anything of that sort," Katya responds somewhat aggressively. "Look, if we want to track down who dropped those bombs then we need to get our hands on better scanning equipment, equipment that the Navy has on their unmanned surveillance drones, and you're the one with a ship unconnected to the Free Worlds. All you need to do is find a lone surveillance drone to plunder. Legally the Navy can't do anything about it, and no one will be hurt in the process. If we succeed in finding who's responsible for the bombings, we could prevent war between the Republic and the Free Worlds, saving thousands or even potentially millions of lives. So are you with us or not?"`
				goto decision
			
			label terrible

> 			`	"Are you saying we're going to need to find a different captain to work with?" says Katya. "If we want to track down who dropped those bombs then we need to get our hands on better scanning equipment, equipment that the Navy has on their unmanned surveillance drones, and you're the one with a ship unconnected to the Free Worlds. All you need to do is find a lone surveillance drone to plunder. Legally the Navy can't do anything about it, and no one will be hurt in the process. If we succeed in finding who's responsible for the bombings, we could prevent war between the Republic and the Free Worlds, saving thousands or even potentially millions of lives. So are you with us or not?"`

Reviewer: "The one" implies that every other ship in the game is connected to the Free Worlds.

```suggestion
			`	"Are you saying we're going to need to find a different captain to work with?" says Katya. "If we want to track down who dropped those bombs then we need to get our hands on better scanning equipment, equipment that the Navy has on their unmanned surveillance drones, and you're the only pilot we can trust whose ship isn't connected to the Free Worlds. All you need to do is find a lone surveillance drone to plunder. Legally the Navy can't do anything about it, and no one will be hurt in the process. If we succeed in finding who's responsible for the bombings, we could prevent war between the Republic and the Free Worlds, saving thousands or even potentially millions of lives. So are you with us or not?"`
```

Author: "you're the one" is referring to the player within the group of them, Katya and Eyes.

Reviewer: True, but couldn't they always find another captain if need be? I think having some allusion to _why_ they don't just get another captain could be helpful. Maybe even:
```suggestion
			`	"Are you saying we're going to need to find a different captain to work with?" says Katya. "If we want to track down who dropped those bombs then we need to get our hands on better scanning equipment, equipment that the Navy has on their unmanned surveillance drones, and we can't exactly go asking around local spaceports to find another pilot to get it for us. All you need to do is find a lone surveillance drone to plunder. Legally the Navy can't do anything about it, and no one will be hurt in the process. If we succeed in finding who's responsible for the bombings, we could prevent war between the Republic and the Free Worlds, saving thousands or even potentially millions of lives. So are you with us or not?"`
```
although that doesn't fit too well with the first line.

Author: The reason Katya and Eyes haven't found another captain yet isn't because they aren't able to. It's because the player is still there and hasn't definitively said no. Katya's speech here isn't about how the player is the only person who can help them, it's about trying to appeal to the player's sense of morality.

Reviewer: In that case, maybe just remove that bit entirely?
"You're the one" sounds like their reasoning is based on it being impossible/difficult to find another captain.

```suggestion
			`	"Are you saying we're going to need to find a different captain to work with?" says Katya. "If we want to track down who dropped those bombs then we need to get our hands on better scanning equipment, equipment that the Navy has on their unmanned surveillance drones. All you need to do is find a lone surveillance drone to plunder. Legally the Navy can't do anything about it, and no one will be hurt in the process. If we succeed in finding who's responsible for the bombings, we could prevent war between the Republic and the Free Worlds, saving thousands or even potentially millions of lives. So are you with us or not?"`
```

			label decision
			choice
				`	"Fine, I'll do it. But I still think it's an unwise idea."`

> 				`	"If this is the sort of reasoning the Free Worlds runs on, then you can kiss goodbye to working with me ever again."`

Reviewer 2: ```suggestion
				`	"If this is the sort of reasoning the Free Worlds runs on, then you can say goodbye to working with me ever again."`
```
"kiss" I think has... connotations.

Author: Not that I'm aware of.

Reviewer 3: "kiss goodbye" is a fairly common English phrase. I'm not aware of any problems associated with using it.

> 				`	"If this is the sort of reasoning the Free Worlds runs on, then you can kiss goodbye to working with me ever again."`

Reviewer: Seems overly aggressive again. My character might be against plundering Navy property, but I can't see myself insulting the mental state of the Free Worlds government in order to do so.

```suggestion
				`	"That sounds like an honorable cause, but it doesn't justify destroying Navy property. Count me out."`
					decline
				`	"If this is the sort of reasoning the Free Worlds runs on, then you can kiss goodbye to working with me ever again."`
```

Author: ```suggestion
				`	"Even still, that doesn't justify destroying Navy property. Count me out."`
					decline
				`	"If this is the sort of reasoning the Free Worlds runs on, then you can kiss goodbye to working with me ever again."`
```

				goto decision
			
			label terrible
			`	"Are you saying we're going to need to find a different captain to work with?" says Katya. "If we want to track down who dropped those bombs then we need to get our hands on better scanning equipment, equipment that the Navy has on their unmanned surveillance drones. All you need to do is find a lone surveillance drone to plunder. Legally the Navy can't do anything about it, and no one will be hurt in the process. If we succeed in finding who's responsible for the bombings, we could prevent war between the Republic and the Free Worlds, saving thousands or even potentially millions of lives. So are you with us or not?"`
			label decision
			choice
				`	"Fine, I'll do it. But I still think it's an unwise idea."`
				`	"Even still, that doesn't justify destroying Navy property. Count me out."`
					decline

> 				`	"If this is the sort of reasoning the Free Worlds runs on, then you can kiss goodbye to working with me ever again."`

Reviewer 4: Do we need two declines for one mission? At least shown in this PR, this seems to be the only place that has more than one.

Reviewer: I think having two adds some valuable role-playing options.

Actually... what if we made the second option set some sort of condition that unlocked some other mission chain (or maybe connected with the Navy campaign later on), before going to the decline in the same way?

				goto decision
			
			label terrible

> 			`	"Are you saying we're going to need to find a different captain to work with?" says Katya. "If we want to track down who dropped those bombs then we need to get our hands on better scanning equipment, equipment that the Navy has on their unmanned surveillance drones, and you're the one with a ship unconnected to the Free Worlds. All you need to do is find a lone surveillance drone to plunder. Legally the Navy can't do anything about it, and no one will be hurt in the process. If we succeed in finding who's responsible for the bombings, we could prevent war between the Republic and Free Worlds, saving thousands or even potentially millions of lives. So are you with us or not?"`

Reviewer 5: ```suggestion
			`	"Are you saying we're going to need to find a different captain to work with?" says Katya. "If we want to track down who dropped those bombs then we need to get our hands on better scanning equipment, equipment that the Navy has on their unmanned surveillance drones, and you're the one with a ship unconnected to the Free Worlds. All you need to do is find a lone surveillance drone to plunder. Legally the Navy can't do anything about it, and no one will be hurt in the process. If we succeed in finding who's responsible for the bombings, we could prevent war between the Republic and the Free Worlds, saving thousands or even potentially millions of lives. So are you with us or not?"`
```

			choice
				`	"Fine, I'm in."`
					accept

> 				`	"I'd rather be shooting down pirates. I'm done with helping the Free Worlds."`

Reviewer: "I'd rather be shooting down pirates" maybe fits the kind of personality that would go the adventure route, but the statement itself is a bit of a non sequitur.
